### PR TITLE
Adding Bottom NavBar

### LINKED ISSUE #10: Bottom Nav Bar
https://pub.dev/packages/google_nav_bar

### PR DESCRIPTION
**Pull request by suntwalriya**
Please check it for the floating part.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,117 +1,108 @@
...
-              'You have pushed the button this many times:',
-            ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headline4,
+        child: _widgetOptions.elementAt(_selectedIndex),
+      ),
+      bottomNavigationBar: 
+      SafeArea(
+        child: Container(
```
Comment (himanshusharma89) [inline lib/main.dart]: Can you try the same implementation as here: https://github.com/sooxt98/google_nav_bar/blob/master/example/lib/main_advance.dart

**Comment by himanshusharma89** [comment]
Also, add a screenshot after doing the changes

**Review by himanshusharma89 — COMMENTED** [review]
@carrisunrio Can you update your code as per the review and also attach a screenshot of the bottom navbar. And pull the code before editing.

File: lib/main.dart (hunk 1 above)
Comment (suntwalriya) [reply lib/main.dart]: Okay i will try this and add the screenshot too

**Pull request merged**